exercises/twofer/tests/two-fer.rs

```diff
@@ -0,0 +1,19 @@
+extern crate twofer;
+use twofer::twofer;
+
+#[test]
+fn empty_string() {
+    assert_eq!(twofer(""), "One for you, one for me.");
```

Comment:
> explain to me that such a comment is actually not necessary for certain reasons (don't confuse the student with unnecessary information, etc., I'll let the person who explains it to me come up with the reason why it's not necessary)

I will consider the following part of https://github.com/exercism/rust/pull/520#issuecomment-386879901 to comprise this explanation:

> thought about this exercise as some kind of second "hello world" and introduction to pattern matching. In my opinion the Result/Some stuff is a more advanced concept and covered in later exercises.

I counter that it is still possible to say as a comment in this test "using the empty string as a sentinel value is not preferred in Rust since we have better ways to express the absence of a value, but we will learn those ways later in the track" without having to specify what those alternative ways are.

I cannot comment on what is the best way to teach students these concepts; I would have to spent time on improving my skills and knowledge before I can effectively spend time to think about and comment on how to teach.

Thus, it does not make sense for me to require any changes before this PR should be merged.